src/main/java/nl/novi/techItEasy/dtos/UserDto.java

```diff
@@ -0,0 +1,65 @@
+package nl.novi.techItEasy.dtos;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import nl.novi.techItEasy.models.Authority;
+
+import java.util.Set;
+
+public class UserDto {
```

Comment:
In deze klasse is het misschien ook handig om validatieregels te zetten. Zo kun je bijvoorbeeld zeker zijn dat er een username en password wordt ingevuld wanneer er een nieuwe user gepost wordt